frontend/src/metabase/query_builder/components/notebook/steps/JoinStep/JoinStep.unit.spec.tsx

```diff
@@ -278,6 +284,39 @@ describe("Notebook Editor > Join Step", () => {
     ).not.toBeInTheDocument();
   });
 
+  it("questions from another database should not appear in recents", async () => {
+    setup(undefined, {
```

Comment:
Awkward `undefined`. Looks like it's time to refactor `setup` to only accept a single param object.